mlbench_core/cli/cli.py

```diff
@@ -113,6 +114,13 @@ def cli(args=None):
 @click.option("--dashboard-url", "-u", default=None, type=str)
 def run(name, num_workers, gpu, light, dashboard_url):
     """Start a new run for a benchmark image"""
+    current_run_inputs = {}
+
+    try:
+        last_run_inputs = pickle.load(open("last_run_inputs.pkl", "rb"))
+    except:
```

Comment:
You need to catch an exception here. `except FileNotFoundError as e`.
Maybe also print a statement saying no previous inputs were found ?

Reply:
I don't think it's necessary to print that no previous inputs were found. In that case, we just offer the existing defaults as we do now.